my-profile-webapp/src/main/webapp/partials/countries.html

```diff
@@ -0,0 +1,251 @@
+<select id="countries" class="form-control" required ng-model="contact.addresses[0].country">
```

Comment:
Yea I did mean to only use the first in the addresses array - do we really need to support multiple addresses for emergency contacts? Even one doesn't seem incredibly needed - in an emergency you're not going to mail someone something. 

We could revise the json structure to only have one address instead or an array, or we can just target the first element of the array. Thoughts?